CHANGELOG.md

```diff
@@ -78,4 +78,4 @@
 
 #### Mix
 
-  * [Mix.Utils] `underscore/1` and `camelize/1` are deprecated
```

Comment:
@josevalim not the final newline, there's an *extra* empty line before the final newline that we always keep. I think the extra line before the final newline should go, it's usually considered trailing whitespace?